.gitignore

```diff
@@ -6,3 +6,5 @@
 /build
 /captures
 .externalNativeBuild
+google-services.json
+GoogleService-Info.plist
```

Comment:
I suppose we should add here debug and release paths

Reply:
No, because other files in those folders may be needed in git.